Add Laravel Livewire framework to tech registry

shakib04 opened the pull request:
## Description

This pull request introduces a new technology entry for **Laravel Livewire**, a full-stack framework for building dynamic interfaces in Laravel, to the tech registry. The new entry has been categorized under "frameworks" and follows the schema defined in `schema.yaml`.

Key changes include:
- A new YAML file `laravel-livewire.yaml` created in the `tech-registry/technologies/frameworks` directory.
- The entry contains all required fields, including `id`, `name`, `description`, `category`, `creation_year`, `owner`, `repo`, `subreddit`, and `stackshare_slug`.

---

## Related Issue

N/A

---

## Type of Change

- [x] ✨ New feature (non-breaking change adding functionality)

---

## Testing Performed

The new entry was validated using the project's validation script:
```bash
cd tech-registry
npm run validate
```
All validations passed successfully with no errors.

---

## Screenshots

![image](https://github.com/user-attachments/assets/ae95b227-a8c7-407e-a8e9-72c515db5a01)

---

## Checklist

- [x] My code follows the code style of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix/feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have checked my code and corrected any misspellings

---

## Tech Insight (Optional)

Adding Laravel Livewire highlights the importance of tracking emerging frameworks that simplify dynamic front-end development while remaining tightly integrated with backend systems.

---

## Your Tech Funeral Speech (Optional)

I bid farewell to Flash, the once-dominant technology for interactive web content, which has now been replaced by modern, accessible, and secure alternatives like HTML5.

--- 

Let me know if you'd like any further adjustments!

jobehi commented on tech-registry/technologies/frameworks/laravel-livewire.yaml:
```diff
@@ -0,0 +1,10 @@
+category: framework
+creation_year: 2019
+owner: livewire
+repo: livewire/livewire
```
```suggestion
repo: livewire
```

jobehi commented on tech-registry/technologies/frameworks/laravel-livewire.yaml:
```diff
@@ -0,0 +1,10 @@
+creation_year: 2019
+owner: livewire
+repo: livewire/livewire
+subreddit: laravel
```
```suggestion
subreddit: LaravelLivewire
```

jobehi commented on tech-registry/technologies/frameworks/laravel-livewire.yaml:
```diff
@@ -0,0 +1,10 @@
+owner: livewire
+repo: livewire/livewire
+subreddit: laravel
+stackshare_slug: laravel-livewire
```
Hello @shakib04, Thank you for your contribution ! 
I checked in stackShare and unfortunately livewire does not exist, so the engine will fail to get any info about its company adoptions. 
We can try to remove this line and see how the score will react to it

shakib04 commented:
Hello @jobehi,

Thank you for the feedback! 🙌 I’ve made the requested changes:

1. Updated the repo field to livewire.
2. Changed the subreddit field to LaravelLivewire.
3. Removed the stackshare_slug field as Laravel Livewire is not listed on StackShare.

Let me know if there’s anything else that needs adjustment!

jobehi reviewed (approved):
Hey @shakib04 LGTM, thank you
